conda/core/prefix_data.py

```diff
@@ -487,10 +483,10 @@ def _load_single_record(self, prefix_record_json_path: PathType) -> None:
         log.debug("loading prefix record %s", prefix_record_json_path)
         with open(prefix_record_json_path) as fh:
             try:
-                json_data = json_load(fh.read())
+                json_data = json.loads(fh.read())
```

Comment:
Why not pass the descriptor instead?

```suggestion
                json_data = json.load(fh)
```

Reply:
loads is faster than load(fh)